cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/command/AbstractGenerateCommand.java

```diff
@@ -95,30 +141,128 @@ public Integer call() throws Exception {
         // Get the correct scm information for builds which have either SUCCESS or NO_REBUILD_REQUIRED status
         String scmUrl = build.getScmUrl();
         String scmTag = build.getScmTag();
+
+        JsonNode bom = null;
+
         if (org.jboss.pnc.enums.BuildStatus.NO_REBUILD_REQUIRED.equals(build.getStatus())) {
```

Comment:
Shouldn't we check here whether `force`  is set and if it is, the generate it either case, without even looking at the possible `NO_REBUILD_REQUIRED` state?

Reply:
Mmmmm I don't think the `force` and `state` are correlated. The `force` is at the moment used to wipe any existing workspace in case is already exists, before the clone operation. I honestly fail to think of how that could happen, given that a new pod is spin up to take care of that for every build? The NO_REBUILD_REQUIRED means than no new build was done at all in PNC, because there were no changes to the build config. So ideally, it would be kind of wrong to regenerate the SBOM from scratch. If the force (as intended in PNC) was used to force a new build anyway, we would not receive the NO_REBUILD_REQUIRED state.

Reply:
OK.

Note about the CLI:

I guess the think is that you need to start thinking about CLI being separate from the service. I should be able to run the CLI, without service. This was the idea. Service is a wrapper/orchestrator around the CLI, but I should be able to do everything in the service, just locally.